src/components/messageFlowButton.vue

```diff
@@ -1,30 +1,23 @@
 <template>
-  <svg
+  <img
+    :src="messageFlow"
     @click="$emit('click')"
     id="message-flow-button"
     data-test="message-flow-button"
     width="19"
     height="20"
-    viewBox="0 0 19 20"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
+    alt="Create a message flow"
```

Comment:
I tried to check but my emulators wouldn't load up. Do Microsoft browser still display the alt text on hover?

Reply:
Good question, I'll try to find out.

Reply:
I can't find many references to this online. One Reddit post from 2018 did link to a (dead) link on Microsoft saying that this behaviour was being removed from their browsers.